substrate/frame/revive/Cargo.toml

```diff
@@ -69,6 +71,8 @@ polkavm-common = { version = "0.26.0" }
 pretty_assertions = { workspace = true }
 secp256k1 = { workspace = true, features = ["recovery"] }
 serde_json = { workspace = true }
+ethereum = { version = "0.18.2", default-features = false, features = ["with-scale"] }
+rlp = {version = "0.6.1", default-features = false }
```

Comment:
nit: The target branch is a bit behind the latest one, could you rebase on the latest changes? 🙏